core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/udf/r/RUDFOpDesc.scala

```diff
@@ -0,0 +1,174 @@
+package edu.uci.ics.texera.workflow.operators.udf.r
+
+import com.fasterxml.jackson.annotation.{JsonProperty, JsonPropertyDescription}
+import com.google.common.base.Preconditions
+import com.kjetland.jackson.jsonSchema.annotations.JsonSchemaTitle
+import edu.uci.ics.amber.engine.architecture.deploysemantics.layer.OpExecInitInfo
+import edu.uci.ics.amber.engine.architecture.deploysemantics.{PhysicalOp, SchemaPropagationFunc}
+import edu.uci.ics.amber.engine.common.virtualidentity.{ExecutionIdentity, WorkflowIdentity}
+import edu.uci.ics.amber.engine.common.workflow.{InputPort, OutputPort, PortIdentity}
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.{LogicalOp, PortDescription, StateTransferFunc}
+import edu.uci.ics.texera.workflow.common.tuple.schema.{Attribute, Schema}
+import edu.uci.ics.texera.workflow.common.workflow.{PartitionInfo, UnknownPartition}
+
+import scala.util.{Success, Try}
+
+class RUDFOpDesc extends LogicalOp {
+  @JsonProperty(
+    required = true,
+    defaultValue =
+      "# Input: Table, port\n" +
+        "# Output/Return: Table-like object that can be split into tuples\n" +
+        "function(input, port) {\n\n" +
+        "}"
```

Comment:
Can R specify output type? If so please add it. Otherwise we can keep it omitted

Reply:
we can ignore this now